install_config/upgrading/manual_upgrades.adoc

```diff
@@ -934,6 +934,11 @@ security rules allowing access to your node's public IP on port 1936:
 
 If this port is not allowed, the router pod will fail to start after
 installation.
+
+An errata has been released which ensures that the router liveness probe uses
+localhost rather than the public interface. This means you no longer need to open
+the firewall rule above. Please see https://access.redhat.com/errata/product/290/ver=3.1/rhel---7/x86_64/RHBA-2016:0293[RHBA-2016:0293] for details
+of that errata.
```

Comment:
I'm not sure if this should be left in or if we should simply removed the entire section. If they're upgrading right now their likelihood of hitting the issue here is near zero as the current packages don't have this problem.